### PR TITLE
Added new choices to the neighbor choice

### PR DESCRIPTION
**Pull request by sphoang**

### CODE REVIEW

**Comment by sphoang** [comment]
Please review this, @casey-collab

**Comment by pmallory** [comment]
@casey-collab can you take a look?

File: english/talk_to_neighbour/talk_to_neighbour.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 I hear you
 I see you
-I'm Listening
\ No newline at end of file
+I'm Listening
+
+As you speak to your neighbor, you see a dark car approach. Two figures emerge, a man and a woman.
+
+[Talk to the man](english/talk_to_neighbor/woman/scully.md)
```
Comment (casey-collab) [inline english/talk_to_neighbour/talk_to_neighbour.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/talk_to_neighbour/talk_to_neighbour.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+[Talk to the woman](english/talk_to_neighbor/man/mulder.md)
```
Comment (casey-collab) [inline english/talk_to_neighbour/talk_to_neighbour.md]: This link is also invalid.

File: english/talk_to_neighbour/man/mulder.md [hunk 3]
```diff
@@ -0,0 +1 @@
+"I'm FBI Special Agent Fox Mulder," the man says in a monotone voice.
```
Comment (casey-collab) [inline english/talk_to_neighbour/man/mulder.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/talk_to_neighbour/woman/scully.md [hunk 4]
```diff
@@ -0,0 +1 @@
+"I'm Special Agent Dana Scully. You may also know me from The Fall or Hannibal."
```
Comment (casey-collab) [inline english/talk_to_neighbour/woman/scully.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sphoang** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**